src/lib/components/Feedback/index.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * @fileoverview Element that handles feedback on posts.
+ */
+import {html} from 'lit-element';
+import {BaseElement} from '../BaseElement';
+import {trackEvent} from '../../analytics';
+import './_styles.scss';
+
+/**
+ * Element that handles feedback on posts to submit to Analytics.
+ *
+ * @extends {BaseElement}
+ * @final
+ */
+class Feedback extends BaseElement {
+  constructor() {
+    super();
+    this.submitted = false;
+    this.submit = this.submit.bind(this);
+  }
+
+  submit(event) {
+    event.preventDefault();
+    const form = new FormData(event.target);
+    const formIterable = Array.from(form.entries());
+    if (!formIterable.length || this.submitted) {
+      return;
+    }
+
+    this.submitted = true;
+    this.requestUpdate();
+
+    formIterable.forEach((entry) => {
+      trackEvent({
+        category: 'Feedback',
+        action: 'submit',
+        label: entry[0],
+        value: Number(entry[1]),
+      });
+    });
+  }
+
+  updated() {
+    const confirmationElement = this.querySelector('.w-feedback__confirmation');
+    if (this.submitted && confirmationElement) {
+      confirmationElement.focus();
+    }
+  }
+
+  render() {
+    // Because we share CSS with the `app.css` we want this to be in the light DOM
+    return html`
+      <details class="w-details" open>
+        <summary class="w-details__summary">
+          <h2 class="w-details__header">Give feedback</h2>
+        </summary>
+        <div
+          class="w-display--flex w-justify-content--center ${!this.submitted &&
+            'hidden'}"
+        >
+          <p
+            class="w-feedback__confirmation w-text--center w-force-focus"
+            tabindex="-1"
+          >
+            Thank you for the feedback!
+          </p>
+        </div>
+        <form
+          class="w-feedback__form ${this.submitted && 'hidden'}"
+          @submit=${this.submit}
+        >
+          <small>All fields optional</small>
+          <div class="web-feedback__rows">
+            <div class="web-feedback__row">
+              <div></div>
+              <div id="yes-label">Yes</div>
+              <div id="no-label">No</div>
+            </div>
+
+            <div class="web-feedback__row">
+              <div id="helpful-label">
+                <label>Was this page helpful?</label>
```

Comment:
Using a `<label>` isn't necessary here because it's not associated with anything. It would need to either wrap a form input, or use its `for=""` attribute to reference the form input to be associated with it. Instead, the outer div's `id` + the use of `aria-lablledby` is what's providing the label for the form inputs. https://web.dev/labels-and-text-alternatives/

IOW, you can remove the `<label>` and leave the text:

```html
<div id="helpful-label">Was this page helpful?</div>
```